samples/detect.v1p3beta1.js

```diff
@@ -0,0 +1,169 @@
+/**
+ * Copyright 2018, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+//CF: change service path, update helpers file to newest library when able.
+
+function localizeObjects(file) {
+  // [START vision object localization]
+  // Imports the Google Cloud client libraries
+  const vision = require('@google-cloud/vision').v1p3beta1;
+  const fs = require('fs');
+
+  // Standard Endpoint
+  const client = new vision.ImageAnnotatorClient();
+  const request = {
```

Comment:
A TODO comment seems to be missing in this function (similar to the one in lines 90-93).

Reply:
+1

As of right now, these 2 samples are missing the TODO placeholder comments for user defined variables:
 - `vision_localize_objects_uri`
 - `vision_handwritten_ocr_uri`

Reply:
I'm sorry, this has been addressed!  I keep pushing changes in editor but they don't seem to be appearing here- these are lines before const client above:   
  /**
   * TODO(developer): Uncomment the following line before running the sample.
   */
  // const fileName = 'Local image file, e.g. /path/to/image.png';
  // Creates a client